one-line-cr-bot.sh

```diff
@@ -541,6 +547,13 @@ else
     cat "$TMP_DATA_FOLDER"/sorted-findings-"$TARGET_COMMIT".txt
 fi
 
+# post comment to PR?
+if [ "$POST_TO_GITHUB_PR" == "true" ]; then
+    touch "$OUTPUT_FILE"
```

Comment:
Why would you touch this file? In case it does not exist? Wouldn't it make more sense to not call the script at all, if the report doesn't exist?

Reply:
If there is no report, you might want to see that as numbers. The python script is (currently) not robust enough to handle the case of that file not being present at all, so the touch line makes sure it exists.